cmd/podman/containers/restore.go

```diff
@@ -67,10 +65,18 @@ func init() {
 	flags.BoolVar(&restoreOptions.IgnoreStaticIP, "ignore-static-ip", false, "Ignore IP address set via --static-ip")
 	flags.BoolVar(&restoreOptions.IgnoreStaticMAC, "ignore-static-mac", false, "Ignore MAC address set via --mac-address")
 	flags.BoolVar(&restoreOptions.IgnoreVolumes, "ignore-volumes", false, "Do not export volumes associated with container")
+
+	publishFlagName := "publish"
```

Comment:
Nit: if you make any other changes, there's no reason to keep this - you can remove the variable and replace `publishFlagName` in the `StringSliceP` with "publish"

Reply:
It is also used in line 75 `restoreCommand.RegisterFlagCompletionFunc(publishFlagName`. This also a copy from `podman run` or create. No problem removing the variable.

Reply:
removed the `publish` String